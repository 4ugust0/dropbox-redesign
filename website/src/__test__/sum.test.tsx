import { render } from '@testing-library/react'
import Link from 'next/link'

it('renders homepage unchanged', () => {
	const { container } = render(<Link href="https://google.com">{1 + 2}</Link>)
	expect(container).toMatchSnapshot()
})
