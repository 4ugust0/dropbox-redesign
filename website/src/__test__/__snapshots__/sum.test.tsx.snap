// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <a
    href="https://google.com"
  >
    3
  </a>
</div>
`;
